### PR TITLE
Feature #127 - Implement business serializer for getting all information

### PR DESCRIPTION
**Pull request by datsiuk-ira**
Implemented business serializer for getting all information

### CODE REVIEW

File: beauty/api/serializers/business_serializers.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
+"""The module includes serializers for Business model."""
+import logging
```
Comment (Misha86) [inline beauty/api/serializers/business_serializers.py:3]: empty line after module docstring

File: beauty/api/serializers/business_serializers.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+
+logger = logging.getLogger(__name__)
+
+
+class BusinessesSerializer(serializers.ModelSerializer):
+    """Serializer for business base fields."""
+
+    name = serializers.CharField(max_length=20)
+    type = serializers.CharField(max_length=100) # noqa
```
Comment (Misha86) [inline beauty/api/serializers/business_serializers.py]: change type to business_type

File: beauty/api/serializers/business_serializers.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    address = serializers.CharField(max_length=500)
+
+    class Meta:
+        """Meta for OwnerBusinessesSerializer class."""
+
+        model = Business
+        fields = ("name", "type", "address")
```
Comment (Misha86) [inline beauty/api/serializers/business_serializers.py]: change type to business_type

File: beauty/api/serializers/business_serializers.py [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+
+class BusinessAllDetailSerializer(serializers.ModelSerializer):
+    """Serializer for specific business."""
+
+    owner_name = serializers.SerializerMethodField()
+    created_at = serializers.ReadOnlyField()
+    address = serializers.CharField(max_length=500)
+
+    def get_owner_name(self, obj):
```
Comment (Misha86) [inline beauty/api/serializers/business_serializers.py]: need docstring

**Review by Misha86 — CHANGES_REQUESTED** [review]
change type to business_type
empty line after module docstring

**Review by pazuzu-ua — CHANGES_REQUESTED** [review]
Good job! Just make those amendments per Mykhailo's suggestions

**Comment by datsiuk-ira** [comment]
Closes #127 #128 #126

**Review by OleksiiDatsiuk — APPROVED** [review]

**Review by morento101 — APPROVED** [review]
<html>
<body>


:sunglasses:
--



</body>
</html>

**Review by pazuzu-ua — APPROVED** [review]

**Pull request merged**